.pre-commit-config.yaml

```diff
@@ -6,3 +6,10 @@ repos:
     - id: go-fmt
     - id: go-build
     - id: go-mod-tidy
+- repo: local
+  hooks:
+    - id: check-license
+      name: check-license
+      entry: scripts/check-license.sh
+      language: script
+      pass_filenames: false
```

Comment:
nit: keep a newline at the end of the file

Reply:
Done, and also I had created https://github.com/Snowflake-Labs/sansshell/pull/475 to avoid such situations in future

Reply:
awesome